### ISSUE TITLE
Create containers with `mkdir`

### ISSUE THREAD

**Issue by wachsylon** [fsspec/swiftspec#7]
It would be nice to have the `mkdir` method at least for the lowest level, the container because it seems container are not created on the fly when a mapper is used:

```
a=fsspec.get_mapper("swift://swift.dkrz.de/dkrz_4236b71e-04df-456b-8a32-5d66641510f2/testtzis/testcmip6")
a[".test"]=b"ho"

404, message='Not Found', url=URL('https://swift.dkrz.de/v1/dkrz_4236b71e-04df-456b-8a32-5d66641510f2/testtzis/testcmip6/.test')
```

Do you think that is possible?

**Comment by d70-t** [comment]
Yes, implementing `mkdir` and `rmdir` on containers should be straight forward (I'd consider it the second level though, because there are `account`s as well. We'd need (from the [SWIFT API](https://docs.openstack.org/api-ref/object-store/)):
* `PUT /v1/{account}/{container}` for `mkdir`
* `DELETE /v1/{account}/{container}` for `rmdir`

I'd probably even go one step further (maybe in another PR) and also catch the error you describe. Then create the container and redo the request. We might want to make this optional though...

I might not be able to that change very soon, but PR's are welcome.
